sled-agent/types/src/rack_ops.rs

```diff
@@ -5,16 +5,17 @@
 use omicron_uuid_kinds::{RackInitUuid, RackResetUuid};
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
+use strum::{EnumCount, IntoEnumIterator};
+use strum_macros::{EnumCount as EnumCountMacro, EnumIter};
```

Comment:
The `strum_macros` are re-exported by strum itself, I thought? Or does something weird happen when you do:

```suggestion
use strum::{EnumCount, EnumIter, IntoEnumIterator};
```

and use those in the `#[derive]` directly?

Reply:
Yeah, that seems to compile, thanks.
Updated in 3c62e025d

Reply:
I had noticed this since `strum_macros` was added as a dep to the workspace Cargo.toml and to sled-agent, so it'd be good to remove those too.

Reply:
That only seems fair.  Done in a3558d4c2